main.go

```diff
@@ -34,6 +34,7 @@ func init() {
 func main() {
 	var (
 		duration = flag.Duration("duration", time.Hour, "The duration that the credentials will be valid for.")
+		format   = flag.String("format", "", "Format can be 'bash' or 'powershell'. Default format is based on operating system.")
```

Comment:
Last comment! I promise!

We should move the `if *format == ""` block into a function, and use that function as the default here. Then the `--help` text will show what the default is set as:

```golang
flag.String("format", defaultFormat(), "Format can be 'bash' or 'powershell'. Default format is based on operating system.")
```

```golang
func defaultFormat() string {
        switch runtime.GOOS {
        case "windows":
                return "powershell"
        default:
                return "bash"
        }
}
```